dashboard.html

```diff
@@ -10,24 +10,39 @@
 <html>
     <head>
         <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <title>My Vineyard - Plantalytics</title>
-        
-        <!-- Includes -->
-        <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet"> 
-        <script src="http://code.jquery.com/jquery-2.0.0.js"></script>
-        <link rel="stylesheet" href="css/dashboard.css"/>
-        <script type="text/javascript" src="js/dashboard.js"></script>
-        <script src="https://maps.googleapis.com/maps/api/js?callback=readyMap" async defer></script>
-        
+
+        <!------------------------------ CDNs ------------------------------>
+
+        <!-- Fonts -->
+        <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+
+        <!-- jQuery -->
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
+
+        <!-- Leaflet -->
+        <link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.css" />
+        <script src="http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.js"></script>
+
         <!-- Favicon Links -->
```

Comment:
This is nitpicky of me, but the Favicons should be grouped with the local files.